src/commands/ban.js

```diff
@@ -2,9 +2,42 @@ const Discord = require("discord.js");
 const errors = require("../utils/errors.js");
 
 module.exports.run = async (bot, message, args) => {
-//stub
+
+    if(!message.member.hasPermission('MANAGE_MESSAGES')){ //check permissions
+        return errors.noPerms(message, "Moderator Role Needed")
+    }
+
+    args = message.content.split(' ').slice(1);
+    const userToBan = message.mentions.users.first();
+    const banReason = args.slice(1).join(' ');
+    
+    if (userToBan) {
+        if (userToBan.length > 2) {
```

Comment:
message.mentions.users.first() returns one user object. So you can't use the length to see if only one member is being mentioned.

Reply:
youre right, ill remove this check